sinch/core/clients/sinch_client_configuration.py

```diff
@@ -21,21 +22,27 @@ def __init__(
         disable_https=False,
         connection_timeout=10,
         application_key: str = None,
-        application_secret: str = None
+        application_secret: str = None,
+        service_plan_id: str = None,
+        sms_api_token: str = None
     ):
         self.key_id = key_id
         self.key_secret = key_secret
         self.project_id = project_id
         self.application_key = application_key
         self.application_secret = application_secret
         self.connection_timeout = connection_timeout
+        self.sms_api_token = sms_api_token
+        self.service_plan_id = service_plan_id
         self.auth_origin = "auth.sinch.com"
         self.numbers_origin = "numbers.api.sinch.com"
         self.verification_origin = "verification.api.sinch.com"
         self._conversation_region = "eu"
         self._conversation_domain = ".conversation.api.sinch.com"
         self._sms_region = "us"
         self._sms_domain = "zt.{}.sms.api.sinch.com"
+        self._sms_domain_with_service_plan_id = "sms.api.sinch.com"
```

Comment:
You may want to offer the possibility to use other regions than the default one with this protocol too.

Reply:
I think this comment was already addressed.

Reply:
In this case, I would like to see a test

Reply:
test added 👌